pkg/session/bootstrap.go

```diff
@@ -2844,6 +2865,7 @@ func upgradeToVer177(s Session, ver int64) {
 	if err != nil {
 		logutil.BgLogger().Fatal("upgradeToVer177 error", zap.Error(err))
 	}
+	writeDDLTableVersion(s)
```

Comment:
Need use a new upgradeVer instead of Ver177.

Reply:
Hi @Defined2014, I have now updated the code and also added an upgrade test.